tests/cases/debugger_hlsl_line_stepping_no_inline_param.amber

```diff
@@ -26,8 +26,7 @@ SHADER vertex vtex_shader SPIRV-ASM
                OpMemoryModel Logical GLSL450
                OpEntryPoint Vertex %main "main" %in_var_POSITION %in_var_COLOR %gl_Position %out_var_COLOR
           %7 = OpString "vs.hlsl"
-               OpSource HLSL 600 %7 "#line 1 \"vs.hlsl\"
-struct VS_OUTPUT {
+               OpSource HLSL 600 %7 "struct VS_OUTPUT {
```

Comment:
The `#line` stuff is just another complication I don't want to deal with right now, so I just removed it.